scripts/a11y/a11y-output.txt

```diff
@@ -0,0 +1,1346 @@
+[ { violations: [],
```

Comment:
This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.